.changeset/cool-trees-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": patch
+---
+
+Refactor: Use class-validator instead of manual validation for ISO date string
```

Comment:
Do we need a changeset for a non-public-API-change?

Reply:
My arguments for this:
- hypothetically, if this is the only change it does not get published, right?
- if something is breaking you might be spotting the change by scanning through the changelog

I think we should have a documented policy for this

Reply:
As discussed: Not required for internal non-breaking changes

Reply:
@johnnyomair removed